packages/dev/core/src/Materials/Node/nodeMaterialDecorator.ts

```diff
@@ -44,7 +44,7 @@ export interface IEditablePropertyOption {
         /** the onPreviewCommandActivated observer of the preview manager should be triggered */
         activatePreviewCommand?: boolean;
         /** a callback to trigger */
-        callback?: (scene: Scene, block: NodeMaterialBlock) => void;
+        callback?: (scene: Scene, block: NodeMaterialBlock) => void | boolean;

```

Comment:
i find it a bit confusing that the callback returns void | boolean. is it not changed to boolean for back-compat?

Reply:
If I just put "boolean", I think it will not be backwards compatible because the existing callbacks will have to be modified to return `false` instead of returning nothing?

Reply:
that's a back-compat issue, i guess. Maybe we can change it to boolean | undefined ?